### DISCUSSION TITLE
How to hide winbar in spesific buffer

### DISCUSSION THREAD

**Discussion by mnabila** [rebelot/heirline.nvim#138] [General]
I have some issues with winbar on Telescope Prompt with ivy theme. when I open telescope, winbar always shows on top of the window.

![image](https://user-images.githubusercontent.com/37516576/231313910-4f085d82-4070-463a-a4df-4f62ec330ff3.png)

**Comment by rebelot** [comment]
You can prevent winbar placement for most plugin windows using the following:

```lua
require("heirline").setup({
   ...
    opts = {
        disable_winbar_cb = function(args)
            local buf = args.buf
            local buftype = vim.tbl_contains({ "prompt", "nofile", "help", "quickfix" }, vim.bo[buf].buftype)
            local filetype = vim.tbl_contains({ "gitcommit", "fugitive", "Trouble", "packer", "dashboard" }, vim.bo[buf].filetype)
            return buftype or filetype
        end,
    },
})
```

**Comment by mnabila** [comment]
Thank you :+1: 

![image](https://user-images.githubusercontent.com/37516576/231418994-367e9d51-47c4-44a9-9f8a-58a536474bf9.png)
